docs/style-guide.md

```diff
@@ -86,7 +86,7 @@ The following requirements are only applicable to your final squashed commit. It
 
 - Use camel-case with capital letters for acronyms, e.g. HTTP, API, etc.
 - Import names and aliases SHOULD be precise. v1 isn't. E.g. in the context of K8s, we use corev1, metav1, appsv1 etc.
-- Be consistent here, too: Match parameters and type names at package level, e.g. single name for a client parameter across different functions. Match import aliases across files and preferably across the whole project. The importas linter can be used to enforce this.
+- Be consistent here, too: Match parameters and type names at package level, e.g. single name for a client parameter across different functions. Match import aliases across files and preferably across the whole project. The imports linter can be used to enforce this.
```

Comment:
`importas` was correct: https://github.com/ionos-cloud/cluster-api-provider-ionoscloud/blob/d6e06821555aa80084b1388f23f53d7807619d1c/.golangci.yml#L25

Maybe add backticks around it. Actually this whole file could need some more backticks.